packages/office-ui-fabric-react/src/components/Image/Image.Props.ts

```diff
@@ -30,6 +30,11 @@ export interface IImageProps extends React.ImgHTMLAttributes<HTMLImageElement> {
   className?: string;
 
   /**
+   * Specifies alternative text for an image.
+   */
+  alt?: string;
```

Comment:
you probably don't need to specify this prop. It's part of `React.ImgHTMLAttributes<HTMLImageElement>` which this interface is extending (just like src, height, width etc).

Reply:
That makes sense - one thing that specifying the prop gives us is the added documentation encouraging it's use. @Jahnp are there other places on the site we can use to show that design guidance encourages alt text, other than adding it explicitly as a prop? The "Best Practices" section is currently unused - could be an option? https://dev.office.com/fabric#Best Practices

Reply:
Yeah, best practices for sure, and definitely in the example. In the future it'd be nice to be able to link out to the type definition being extended.